pipeline/4/jenkinsfile-tier-1-object.groovy

```diff
@@ -0,0 +1,76 @@
+/*
+    Pipeline script for executing Tier 1 object test suites for RH Ceph 5.0.
```

Comment:
The document string points to 5.0. Please change it to 4.x

Reply:
Done